packages/core/test/popover/popoverTests.tsx

```diff
@@ -396,7 +396,7 @@ describe("<Popover>", () => {
 
             wrapper.simulateTarget("click").assertIsOpen();
 
-            TestUtils.Simulate.click(document.getElementsByClassName(Classes.POPOVER_DISMISS)[0]);
+            Simulate.click(document.getElementsByClassName(Classes.POPOVER_DISMISS)[0]);
```

Comment:
we must use TestUtils.Simulate on DOM elements outside of enzyme wrappers. this is basically required for Portal-ed components because enzyme + react 15 do not support portals well. https://github.com/airbnb/enzyme/issues/252